biotracks/readfile.py

```diff
@@ -326,19 +280,7 @@ def read_file(f, track_dict):
     """
     # check for file extension
     if f.endswith('.xls'):
-        # first thing, try to convert it to a plain csv
-        try:
-            xls_to_csv(f)
-            name, extension = os.path.splitext(f)
-            f = name + '.csv'
-            print('XLS converted into csv...')
-        except XLRDError:
-            # copy the file and save it as csv
-            import shutil
-            name, extension = os.path.splitext(f)
-            shutil.copyfile(f, name + '.csv')
-            f = name + '.csv'
-            print('Not an excel file.' + ' Copied and simply renamed to csv.')
+        objects, links = read_icy(f)
```

Comment:
Done in 6999518, but let's keep in mind that we should move from prints to using the logging module.